frontend/utilities/strings/stringUtils.ts

```diff
@@ -79,9 +79,25 @@ export const strToBool = (str?: string | null) => {
   return str ? JSON.parse(str) : false;
 };
 
+export const stripQuotes = (string: string) => {
+  // Regular expression to match quotes at the start and end of the string
+  const quoteRegex = /^["''""]+([\s\S]*?)["''""]$/;
```

Comment:
I think `["']` does the same thing. Also, if we do want to match both single and double quoted strings, do we want to make sure the match has the same multiplicity for opening/closing? So, "matches", 'matches', "no match', 'no match"

Reply:
Yeah, there's a check for `isIncompleteQuoteQuery` for when the user is typing. That checks the start end end quote are the same. It isn't until they are that the query is then sent to this `stripQuotes` function to call the `/vulnerabilities/:cve` as the `:cve` so the check for matching quotes is earlier on.

Reply:
`stripQuotes` accounts for all of the examples you gave: "matches", 'matches', "no match', 'no match"

Reply:
Wild! I am either missing something, or that AI is wrong:
<img width="680" alt="Screenshot 2024-08-27 at 10 53 04 AM" src="https://github.com/user-attachments/assets/908356a8-c178-477f-954f-828d658fa547">

Reply:
Appreciate you, PR incoming

Reply:
https://github.com/fleetdm/fleet/pull/21606